test/users/test-user.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Do our YAML files need copyright notices? I looked, and some of our YAML files have the copyright notice and some of them do not.

Reply:
I don't see any YAML files with copyright notices.  Where did you find them?

Reply:
We can discuss this at the next KubeArchive meeting.

Reply:
I created this related issue -> https://github.com/kubearchive/kubearchive/issues/462